<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolkitDetails</class>
 <widget class="QWidget" name="PolkitDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>90</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="appLabel">
     <property name="text">
      <string>Application:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="actionLabel">
     <property name="text">
      <string>Action:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="vendorLabel">
     <property name="text">
      <string>Vendor:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLabel" name="vendorUrlLabel">
     <property name="text">
      <string>Vendor:</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
     <property name="tipText" stdset="0">
      <string/>
     </property>
     <property name="useTips" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLabel" name="actionTextLabel">
     <property name="text">
      <string>Action:</string>
     </property>
     <property name="tipText" stdset="0">
      <string/>
     </property>
     <property name="useTips" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLabel" name="appTextLabel">
     <property name="text">
      <string>Application:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
